src/main/java/com/synopsys/integration/alert/channel/jira/JiraIssueHandler.java

```diff
@@ -186,12 +187,23 @@ public String createOrUpdateIssues(final FieldAccessor fieldAccessor, final Mess
                     }
                 }
             } catch (JiraMissingTransitionException e) {
-                missingTransitions.add(e.getTransition());
+                if (missingTransitionToIssues.containsKey(e.getTransition())) {
```

Comment:
Our code is really getting bloated in this class. Are there any abstractions we can make around the error handling to cut down on this?